Source/JavaScriptCore/runtime/ModuleProgramExecutable.cpp

```diff
@@ -41,31 +41,37 @@ ModuleProgramExecutable::ModuleProgramExecutable(JSGlobalObject* globalObject, c
         vm.functionHasExecutedCache()->insertUnexecutedRange(sourceID(), typeProfilingStartOffset(vm), typeProfilingEndOffset(vm));
 }
 
-ModuleProgramExecutable* ModuleProgramExecutable::create(JSGlobalObject* globalObject, const SourceCode& source)
+UnlinkedModuleProgramCodeBlock* ModuleProgramExecutable::generateOrGetUnlinkedCodeBlock(JSGlobalObject* globalObject, ThrowScope& throwScope, ParserError& error)
```

Comment:
Do not take `ThrowScope& throwScope`. If the function throws an error, you should define in as `auto scope = DECLARE_THROW_SCOPE(vm)` instead.